tests/bootstrap.php

```diff
@@ -532,4 +538,20 @@ function getItemByTypeName($type, $name, $onlyid=false) {
    return false;
 }
 
+/**
+ * Test helper, search an component from its type and designation
+ *
+ * @param string  $type
+ * @param string  $name
+ * @param boolean $onlyid
+ * @return the item, or its id
+ */
+function getItemByTypeDesignation($type, $name, $onlyid=false) {
```

Comment:
It would be better I guess to use existing `getItemByTypeName()` making it relying on `$item::getnameField()` instead of hardcoced "name" field.